### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by developer-anand**
Added card component

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Looks good fr me.

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  margin: 1rem;
+  padding: 1rem;
+  display: flex;
+  width: auto;
+  max-width: 25rem;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  border: 1px solid black;
```
Comment (zoef-yt) [inline components/cards/cards.css]: use var for colour

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
+@import "/common.css";
+
+.card-title {
+  font-size: 3rem;
+  font-weight: 600;
+  margin: 1rem;
+}
```
Comment (zoef-yt) [inline components/cards/cards.css:7]: add spacing between line 7 and 8

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+.fa:hover {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+}
+
+.card-overlay {
+  position: absolute;
+  color: var(--secondary-color);
+  background-color: rgba(62, 67, 75, 0.5);
```
Comment (zoef-yt) [inline components/cards/cards.css]: use var for colour

File: components/cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+.card-btn-text {
+  display: flex;
+  font-size: 4rem;
+  color: var(--primary-color);
+  background-color: var(--black);
+  align-items: center;
+}
+
+.card-horizontal {
+  border: 1px solid black;
```
Comment (zoef-yt) [inline components/cards/cards.css]: use var for colour

File: components/cards/cards.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+.card-horizontal-content {
+  margin-top: 4rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.shadow {
+  box-shadow: 10px 10px 5px 0px rgb(0 0 0 / 30%);
```
Comment (zoef-yt) [inline components/cards/cards.css]: use var for colour

File: components/cards/cards.html [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+            <!--
+            style="
+              width: 646px;
+              height: 347px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
+            --
+          >
```
Comment (zoef-yt) [inline components/cards/cards.html]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by DivyanshSareen — APPROVED** [review]
do the changes asked by zoef and you are good to merge

**Comment by developer-anand** [comment]
> do the changes asked by zoef and you are good to merge

All changes done.

**Review by zoef-yt — APPROVED** [review]
Good to merge. Good job.

**Pull request merged**